src/store/trees/trees.js

```diff
@@ -50,12 +51,14 @@ export default {
         enabledFeatures: [
           /** @see \Voto5\Http\Controllers\V3TreesController::get_editTree */
         ],
+        saveCurrentlyInProgress: 0,
+        isEditable: true
```

Comment:
Can we rename our `created` property to `isCreated` / `is_created` to indicate boolean? I'd initially read that as a date/timestamp, equivalent to something like `createdAt`. 

`saveCurrentlyInProgress` → `isSaveCurrentlyInProgress`
`configured` → `isConfigured`

Reply:
Done